### ISSUE TITLE
Combination of CURRENT variables + WRAPPERS is not working

### ISSUE THREAD

**Issue by kinow** [BSC-ES/autosubmit#1238]
In GitLab by @ainagaya on Mar 4, 2024, 17:39

Hello @dbeltrankyl and @kinow,

### Autosubmit Version

autosubmit/4.0.104

### Expid affected(If applicable)

a0up

### Which task has issues? Where is the log(If applicable)

* **Full_name**: a0up_ASThread_17095535864709_1_1.cmd

* **Log_Path:** /appl/AS/AUTOSUBMIT_DATA/a0up/tmp/a0up_ASThread_17095535864709_1_1.cmd

### Summary

I wanted to add horizontal wrappers to my transfer jobs for destinE. Each job uses just one processor, and LUMI is not really good at sharing nodes, so I wanted to "force" that a bunch of jobs went to the same node. This is my wrapper definition:

```
WRAPPERS:
        WRAPPER_0:
                TYPE: "horizontal"
                JOBS_IN_WRAPPER: "TRANSFER"
                MAX_WRAPPED: 100
```

I checked the monitor and it looked cool. But when I submitted, I saw that the --partition was not being parsed. I'm using:

```
PLATFORMS:
  LUMI:
    APP_PARTITION: "ju-strategic"
```

In the normal (without wrappers) this is parsed correctly. But in the wrapper is not being substituted: 

```

#SBATCH -J a0up_ASThread_17095535864709_1_1
#
#SBATCH --partition=%CURRENT_APP_PARTITION%
#
#SBATCH -A project_465000454
#SBATCH --output=a0up_ASThread_17095535864709_1_1.out
#SBATCH --error=a0up_ASThread_17095535864709_1_1.err
#SBATCH -t 03:00:00
```

I copied my experiment a0up to a0zt with the current state, because I will unset the usage of %CURRENT_APP_PARTITION% and specify the partition I'll use under JOBS, but I wanted to point this for the future, as it can be important when adding wrappers to OPA/APPS (@franra9 ).

### Steps to reproduce

You can copy a0zt :)

### What is the current bug behavior?

#SBATCH --partition=%CURRENT_APP_PARTITION%

### What is the expected correct behavior?

#SBATCH --partition=ju-strategic

### Any other relevant information(if applicable)

()

**Comment by kinow** [comment]
In GitLab by @dbeltrankyl on Apr 18, 2024, 08:38

v4.1.3 had current_ keywords added to wrappers, 

v4.1.4 will fixes some additional issues where the jobs inside wrappers could not have their parameters updated to the last info

**Comment by kinow** [comment]
In GitLab by @dbeltrankyl on Jun 11, 2024, 11:34

This should be fixed in 4.1.7, if not reopen this issue ( as the milestone was not set to a specific version )
